tech/languages/ocaml/ocaml-installation.md

```diff
@@ -0,0 +1,50 @@
+---
+title: OCaml
+subsection: ocaml
+section: tech-languages
+order: 1
+version: 1.0
+description: An open source multi-paradigm programming language and toolchain. 
+---
+
+# What is OCaml?
+
+Quoting the [ocaml.org](https://ocaml.org/) website:
+
+> OCaml is an industrial-strength programming language supporting functional, imperative and object-oriented styles.
```

Comment:
```suggestion
OCaml is an industrial-strength programming language supporting functional, imperative and object-oriented styles.
```

I don't think a basic descripton can be in any way copyrighted. Especially if it's in Fedora (the license includes these texts).

Reply:
Ok, can I just remove the Quoting? Or prefer I rather change the description for a new one?

Reply:
Right figured out by your change, sorry.